pkg/controller/controller_test.go

```diff
@@ -45,3 +52,54 @@ func TestConvert2SealedSecretPassThrough(t *testing.T) {
 		t.Fatalf("got %v want %v", got, want)
 	}
 }
+
+func TestDefaultConfigDoesNotSkipRecreate(t *testing.T) {
+	ns := "some-namespace"
+	var tweakopts func(*metav1.ListOptions)
+	clientset := fake.NewSimpleClientset()
+	ssc := ssfake.NewSimpleClientset()
+	keyRegistry := testKeyRegister(t, context.Background(), clientset, ns)
+
+	ctrl, got := prepareController(clientset, ns, tweakopts, &Flags{SkipRecreate: false}, ssc, keyRegistry)
```

Comment:
Just semantics: I feel like it's more intuitive to call it this way:

```suggestion
	got, err := prepareController(clientset, ns, tweakopts, &Flags{SkipRecreate: false}, ssc, keyRegistry)
```

If there's an error we fail the tests, otherwise we check what we got.

Reply:
ok, will change it

Reply:
fixed